react-backend/shared/models/cb-admin.js

```diff
@@ -0,0 +1,32 @@
+const then = new Date('2000-01-01T00:00:00.000Z');
+const now = new Date(Date.now());
+
+module.exports = {
+  getFeedback: async (dbConnection, { cbId, since = then, until = now }) => {
+    const getFeedbackQuery = `SELECT feedback_score, feedback_date FROM feedback WHERE cb_id = $1 AND feedback_date BETWEEN $2 AND $3`;
+    const query = await dbConnection.query(getFeedbackQuery, [
+      cbId,
+      since,
+      until,
+    ]);
+    return query.rows;
+  },
+
+  postFeedback: async (dbConnection, { cbEmail, feedbackScore }) => {
```

Comment:
Get rid of `post` here. The DB layer is not related to the API layer. There's no reason to introduce a coupling in the naming of methods and the endpoints on which they're used. My use of `get` in `getFeedback` was not a reference to the HTTP `GET` method. 

Instead, name the method for what it does in the context of the model. It adds a new feedback record. What might be a better name?

Reply:
Lol, I cannot help but read this in a primary school teacher voice...